spec/features/bulk_updates/bulk_updates_location_keywords_spec.rb

```diff
@@ -83,9 +83,10 @@
 
       context 'when viewing the collection' do
         before do
-          within '#bulk-update-status-table' do
-            click_on @find_and_remove_concept_response.body['EntryTitle']
-          end
+          # within '#bulk-update-status-table' do
```

Comment:
remove

also wondering why you switched from clicking on something to using visit?

Reply:
Apologies if this gets a bit confusing: 
So the `before` blocks with `bulk_update_step_1: true` and  `bulk_update_step_2: true` run in those first 2 `it` blocks that specify those options and they already run the bulk update. The `it` block for this step doesn't have those options so it skips those steps, but the bulk update operation is already executed from the other `it` blocks being run first. Here were are just verifying the operation ran correctly by checking the collection, so we don't need them to run again. Also, if those `before` blocks were run again for this `it` block, we would run into an issue with the unique name constraint (which is why these tests were failing) _and_ for the science keyword facets some options would already have been removed and not selectable.

Reply:
👍